avm/res/document-db/database-account/tests/e2e/perimeter/perimeter.bicep

```diff
@@ -0,0 +1,40 @@
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the perimeter.')
+param perimeterName string
+
+@description('Required. The resource ID of the storage account to secure with the perimeter.')
+param cosmosDbResourceId string
+
+var profileName = 'Default'
+
+module perimeter '../../../../../network/network-security-perimeter/main.bicep' = {
```

Comment:
Test should not have any local references to other modules as this opens the doors for breaking changes. Please change to a reference of the published module

Reply:
And I guess when you do, you can remove this file and invoke the module directly in the `main.test.bicep`.

I anyways understand this is like a post-deployment test?